common/scripted_triggers/wc_maa_related_triggers.txt

```diff
@@ -0,0 +1,270 @@
+		
+		land_skirmish_culture_trigger = yes
+		taur_culture_trigger = yes
+	}
+}
+
+### Race size triggers
+is_giant_culture_trigger = {
+	OR = {
+		#Races that big enough to be considered as "giant"
+		has_cultural_pillar = heritage_shathyar
+		has_cultural_pillar = heritage_tharian
+		this = culture:storm_giant
+		this = culture:titanforged
+		this = culture:frozen_giant
+		this = culture:annihilan
+	}
+}
+is_large_culture_trigger = {
+	OR = {
+		#Races that big enough to be considered as "large"
+		has_bovine_heritage_trigger = yes
+		has_cultural_pillar = heritage_draconic
+		has_cultural_pillar = heritage_vrykulic
+		has_cultural_pillar = heritage_gorian
+		has_cultural_pillar = heritage_demonic
+		this = culture:titanforged
+		this = culture:mogu
+		this = culture:drogbar
+	}
+	NOT = {
+		this = culture:annihilan
+		this = culture:eredar
+		this = culture:sayaadi
+	}
+}
+is_small_culture_trigger = {
+	OR = {
+		#Races that small enough to be considered as "small"
+		has_cultural_pillar = heritage_gnomish
+		has_cultural_pillar = heritage_amphibian
+		has_cultural_pillar = heritage_murine
+		has_cultural_pillar = heritage_vulpine
+		has_cultural_pillar = heritage_pygmy
+		has_goblin_heritage_trigger = yes
+		this = culture:grummle
+		this = culture:wolvar
+		this = culture:mechagnome
+	}
+}
+
+
+### Can unlock MaA tech
+
+can_unlock_culture_aerial_maa_trigger = {
+	OR = {
+		can_unlock_innovation_naga_army_trigger = yes
+		can_unlock_innovation_obsidian_destroyer_trigger = yes
+		can_unlock_innovation_night_elf_aerial_trigger = yes
+		can_unlock_innovation_gyrocopter_trigger = yes
+	}
+}
+can_unlock_culture_massive_aerial_maa_trigger = {
+	OR = {
+		can_unlock_innovation_night_elf_aerial_trigger = yes
+		can_unlock_innovation_cloud_serpents_trigger = yes
+		can_unlock_innovation_dragon_trigger = yes
+		can_unlock_innovation_scourge_aerial_trigger = yes
+		can_unlock_innovation_proto_drake_rider_trigger = yes
+		can_unlock_innovation_goblin_air_trigger = yes
+	}
+}
+can_unlock_culture_giant_maa_trigger = {
+	OR = {
+		can_unlock_innovation_undead_tactics_trigger = yes
+		can_unlock_innovation_troll_tactics_trigger = yes
+		can_unlock_innovation_drakonid_trigger = yes
+		can_unlock_innovation_golem_arcane_trigger = yes
+		can_unlock_innovation_golem_mech_trigger = yes
+		can_unlock_innovation_demon_tactics_trigger = yes
+		can_unlock_innovation_titan_tactics_trigger = yes
+		can_unlock_innovation_night_elf_ground_trigger = yes
+		can_unlock_innovation_naga_army_trigger = yes
+		can_unlock_innovation_goblin_mechanics_trigger = yes
+		can_unlock_innovation_silithid_colossus_trigger = yes
+		can_unlock_innovation_gnome_mechanics_trigger = yes
+	}
+}
+
+can_unlock_innovation_elephantry_trigger = {
+	landing_of_exodar_happened_trigger = yes
+	taur_culture_trigger = no
+	winged_culture_trigger = no 
+	NOT = {
+		this = culture:naga
+	}
+}
+
+### Has MaA tech (Used for buildings atm)
+
+has_unlocked_special_aerial_maa_trigger = {
```

Comment:
May also create `has_unlocked_aerial_maa_trigger`. The trigger that inludes both special and basic MaAs and use it in buildings.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/189973859-b76c070b-f997-467f-a24d-b8388adfa68e.png)

</details>

Reply:
Ok